scipy/stats/tests/test_distributions.py

```diff
@@ -77,6 +77,9 @@ def test_distributions_submodule():
 
 
 class TestVonMises:
+    def setup_class(self):
```

Comment:
```suggestion
    def setup_method(self):
```
Here and throughout.